dask/dataframe/io/parquet.py

```diff
@@ -1235,5 +1195,39 @@ def to_parquet(df, path, engine='auto', compression='default', write_index=None,
     return out
 
 
+def sorted_columns(statistics):
```

Comment:
Some helper functions are hidden, some are not. Perhaps all functions should be normal and documented?